File: src/main/java/me/diax/diax/commands/fun/Flip.java
package me.diax.diax.commands.fun;

import me.diax.comportment.jdacommand.Command;
import me.diax.comportment.jdacommand.CommandAttribute;
import me.diax.comportment.jdacommand.CommandDescription;
import me.diax.diax.util.Emote;
import net.dv8tion.jda.core.entities.Message;

import java.util.Random;

@CommandDescription(
        name = "flip",
        triggers = {
                "flip", "coin", "heads", "toss", "tails", "hot"
        },
        attributes = {
                @CommandAttribute(key = "private"),
        }
)
public class Flip implements Command {

    @Override
    public void execute(Message message, String s) {
        message.getChannel().sendMessage(Emote.BUTTON + " - You flipped a " + (new Random().nextInt(2) == 1 ? "heads" : "tails") + ".").queue();
    }
}

File: src/main/kotlin/me/diax/diax/commands/fun/Flip.kt
package me.diax.diax.commands.`fun`

import me.diax.comportment.jdacommand.Command
import me.diax.comportment.jdacommand.CommandDescription
import me.diax.diax.util.Emote.DIAXCOIN_HEADS
import me.diax.diax.util.Emote.DIAXCOIN_TAILS
import net.dv8tion.jda.core.entities.Message
import java.util.*

@CommandDescription(
    name = "flip",
    triggers = ["flip", "coin", "heads", "toss", "tails", "hot"]
)
class Flip : Command {
    val results = arrayOf("$DIAXCOIN_HEADS - You flipped a heads.", "$DIAXCOIN_TAILS - You flipped a tails.")

    override fun execute(message: Message, s: String) {
        message.channel.sendMessage(results[Random().nextInt(1)]).queue()
    }
}